src/styles/calendar_box.scss

```
@@ -0,0 +1,31 @@
+.calendar-box {
+  padding-top: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  width: ($square-size + $margin) * 7;
+  display: flex;
+  flex-wrap: wrap;
+  &__day::before {
+    box-sizing: border-box;
+    width: $square-size;
+    height: $square-size;
+    margin-left: $margin;
+    margin-top: $margin;
+    background-color: #eee;
+    display: flex;
+    padding: 10px;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid black;
+    transition: transform 0.5s 0.1s linear;
```

why do you have delay here? also `linear` is default value for easing function
```suggestion
    transition: transform 0.5s;
```

---

what about background color?